ToolBox/IQFeed/IQFeedDataQueueHandler.cs

```diff
@@ -283,7 +283,7 @@ private bool CanSubscribe(Symbol symbol)
             var market = symbol.ID.Market;
             var securityType = symbol.ID.SecurityType;
 
-            if (symbol.Value.ToLower().IndexOf("universe") != -1) return false;
+            if (symbol.Value.ToLowerInvariant().IndexOfInvariant("universe") != -1) return false;
```

Comment:
Maybe this could directly call `IndexOfInvariant(ignoreCase: true), skipping `ToLowerInvariant()`